packages/utilities/psammead-styles/src/__snapshots__/global-styles.test.jsx.snap

```diff
@@ -4,213 +4,248 @@ exports[`global-styles should render global styles 1`] = `
 <head>
   <style
     data-emotion="css-global"
+    data-s=""
```

Comment:
72 line changes in this snapshot are additions of this blank `data-s` property .

The `sourceMappingUrl` is also being removed for every entry - which I think is a good thing. This was added as a negative side effect of the initial emotion migration